smart-contracts/scripts/setters/unlock-oracle-all-tokens.js

```diff
@@ -0,0 +1,30 @@
+const { ethers } = require('hardhat')
+const { networks } = require('@unlock-protocol/networks')
+const setOracle = require('./unlock-oracle')
+
+async function main() {
+  const { chainId } = await ethers.provider.getNetwork()  
+  const {tokens} = networks[chainId]
+
+  for (const token of tokens) {
+    if(token.symbol !== 'WETH') {
```

Comment:
Why the exception for `WETH`?

Reply:
Uniswap will revert when tokens are identical (we are asking for price in WETH)